### PR TITLE
Merge remote-tracking branch 'origin/branch-22.02' into main [skip ci]

### PR DESCRIPTION
**Pull request by NvTimLiu**
Signed-off-by: Tim Liu <timl@nvidia.com>

Note: **Create a new merge request** for the PR's merge

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

File: docs/additional-functionality/rapids-udfs.md [hunk 1]
```diff
@@ -162,11 +162,11 @@ between two float vectors using [native code](../../udf-examples/src/main/cpp/sr
 
 - [URLDecode](../../udf-examples/src/main/java/com/nvidia/spark/rapids/udf/hive/URLDecode.java)
 implements a Hive simple UDF using the
-[Java APIs of RAPIDS cudf](https://docs.rapids.ai/api/cudf-java/stable)
+[Java APIs of RAPIDS cudf](https://docs.rapids.ai/api/cudf-java/legacy)
```
Comment (GaryShen2008) [inline docs/additional-functionality/rapids-udfs.md:165]: This is not the correct link.

**Comment by NvTimLiu** [comment]
build

File: docs/additional-functionality/rapids-udfs.md (hunk 1 above)
Comment (NvTimLiu) [reply docs/additional-functionality/rapids-udfs.md]: fixed by https://github.com/NVIDIA/spark-rapids/pull/4771

**Review by GaryShen2008 — APPROVED** [review]

**Pull request merged**